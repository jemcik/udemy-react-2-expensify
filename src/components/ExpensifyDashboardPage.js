import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesSummary from './ExpensesSummary';
import ExpenseListFilter from './ExpenseListFilter';

const ExpensifyDashboardPage = () => (
    <div>
        <ExpenseListFilter />
        <ExpensesSummary />
        <ExpenseList />
    </div>
);

export default ExpensifyDashboardPage;
