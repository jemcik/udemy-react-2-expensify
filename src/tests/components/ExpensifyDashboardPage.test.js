import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage';

test('dashboard page', () => {
	const wrapper = shallow(<ExpensifyDashboardPage />);

	expect(wrapper).toMatchSnapshot();
});
