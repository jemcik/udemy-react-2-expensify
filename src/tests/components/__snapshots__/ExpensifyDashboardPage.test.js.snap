// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard page 1`] = `
<div>
  <Connect(Component) />
</div>
`;
